ksql-functional-tests/src/test/resources/expected_topology/5_3_0/sink-partitions-replicas_-_should_copy_partition_and_replica_count_from_source_topic

```diff
@@ -15,14 +16,18 @@
   "ksql.internal.topic.replicas" : "1",
   "ksql.insert.into.values.enabled" : "true",
   "ksql.streams.default.production.exception.handler" : "io.confluent.ksql.errors.ProductionExceptionHandlerUtil$LogAndFailProductionExceptionHandler",
+  "ksql.access.validator.enable" : "auto",
   "ksql.streams.bootstrap.servers" : "localhost:0",
   "ksql.streams.commit.interval.ms" : "2000",
   "ksql.streams.auto.commit.interval.ms" : "0",
   "ksql.sink.replicas" : null,
+  "ksql.metrics.extension" : null,
   "ksql.streams.topology.optimization" : "all",
   "ksql.streams.num.stream.threads" : "4",
+  "ksql.metrics.tags.custom" : "",
   "ksql.query.fields.key.legacy" : "false",
   "ksql.udfs.enabled" : "true",
+  "ksql.connect.registry.url" : "http://localhost:8083",
```

Comment:
What are these new configs for?

Reply:
script generating issue... I'll back this out